src/review_heatmap/heatmap.py

```diff
@@ -61,20 +61,37 @@ class HeatmapCreator(object):
     def compress_levels(colors, indices):
         return [colors[i] for i in indices]
 
+    MINUTE = 60 * 1000  # milliseconds
+    DAY = 24 * 60 * MINUTE  # milliseconds
+
     stat_levels = {
         # tuples of threshold value, css_colors index
         "streak": list(zip((0, 14, 30, 90, 180, 365),
                        compress_levels(css_colors, (0, 2, 4, 6, 9, 10)))),
         "percentage": list(zip((0, 25, 50, 60, 70, 80, 85, 90, 95, 99),
                            css_colors)),
+        "time_minute": list(
+            zip(
+                (0, 30 * MINUTE, 40 * MINUTE, 50 * MINUTE, 60 * MINUTE, 70 * MINUTE),
+                compress_levels(css_colors, (0, 2, 4, 6, 9, 10)),
+            )
+        ),
+        "time_day": list(
+            zip(
+                (0, DAY, 3 * DAY, 10 * DAY, 30 * DAY, 100 * DAY),
+                compress_levels(css_colors, (0, 2, 4, 6, 9, 10)),
+            )
```

Comment:
It seems like there's a closing `)` missing here, which currently causes:

```
When loading '⁨review_heatmap⁩':
⁨Traceback (most recent call last):
  File "/home/ari/Projects/AnkiDesktop21/qt/aqt/addons.py", line 208, in loadAddons
    __import__(addon.dir_name)
  File "/home/ari/Tests/Anki/review-heatmap/addons21/review_heatmap/__init__.py", line 118, in <module>
    initializeAddon()
  File "/home/ari/Tests/Anki/review-heatmap/addons21/review_heatmap/__init__.py", line 109, in initializeAddon
    from .views import initializeViews
  File "/home/ari/Tests/Anki/review-heatmap/addons21/review_heatmap/views.py", line 53, in <module>
    from .heatmap import HeatmapCreator
  File "/home/ari/Tests/Anki/review-heatmap/addons21/review_heatmap/heatmap.py", line 85
    }
    ^
SyntaxError: closing parenthesis '}' does not match opening parenthesis '(' on line 79
```

Reply:
sorry about that. I had customized an earlier version of the add-on, and had to deal with a number of conflicts. Didn't properly test this version.